specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/ConnectToTargetSqlSqlDbSyncTask.json

```diff
@@ -0,0 +1,39 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Data Migration Service Resource Provider",
+    "version": "2018-07-15-preview"
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "ConnectToTargetSqlSqlDbSyncTaskInput": {
+      "type": "object",
+      "description": "Input for the task that validates connection to SQL DB and target server requirements",
+      "properties": {
+        "sourceConnectionInfo": {
+          "description": "Connection information for Source SQL Server",
```

Comment:
nit: source instead of Source, as below we say target. Both should be consistent

Reply:
Done.